ubuntuStation/views.py

```diff
@@ -1,54 +1,44 @@
 from ubuntuStation import app
-from flask import redirect
-from flask import render_template
-from flask import request
-from flask import url_for
+from flask import redirect, url_for, session, request, render_template
 from . import login_manager
-from . import limiter
-from flask_login import LoginManager, UserMixin, login_required, login_user, logout_user
-import urllib.parse as urlparse
+from flask_login import login_required, login_user, logout_user, current_user
+import json
 from .forms import *
 from .models import *
 from .database import db_session
+from requests_oauthlib import OAuth2Session
+from .database import data
 
-'''
-@login_manager.request_loader
-def load_user(request):
-    token = request.headers.get('Authorization')
-    if token is None:
-        token = request.args.get('token')
-
-    if token is not None:
-        username,password = token.split(":")
-        user_entry = User.get(username)
-        if (user_entry is not None):
-            user = User(user_entry[0], user_entry[1])
-            if (user.password == password):
-                return user
-    return None
-'''
-class UserManager(UserMixin):
-    user_database = {}
-    users = User.query.all()
-    for u in users:
-        user_database[u.username] = (u.username, u.password_hash)
+class Auth:
+    CLIENT_ID = data["CLIENT_ID"]
+    CLIENT_SECRET = data["CLIENT_SECRET"]
+    REDIRECT_URI = data["REDIRECT_URI"]
+    AUTH_URI = data["AUTH_URI"]
+    TOKEN_URI = data["TOKEN_URI"]
+    USER_INFO = data["USER_INFO"]
+    SCOPE = data["SCOPE"]
 
-    def __init__(self, username):
-        self.id = username
-        self.username = username
-
-    @classmethod
-    def get(cls,id):
-        return cls.user_database.get(id)
+class Config:
+    APP_NAME = "Survey Admin "
+    SECRET_KEY = "development"
```

Comment:
@ryantanaka @jygh98 is this safe to be here? If not, need to get a new secret key and follow the same procedure we did for the other information.